### ISSUE TITLE
crash upon Number.MAX_VALUE.toString(16)

### ISSUE THREAD

**Issue by alexlamsl** [nodejs/node-chakracore#509]
* **Version**: v8.10.0
* **Platform**: Windows 10 x64

#### Node.js
```
$ node -v
v8.11.1

$ echo Number.MAX_VALUE.toString(16);console.log("Done") | node
Done

$
```
#### node-chakracore
```
$ node -v
v8.10.0

$ echo Number.MAX_VALUE.toString(16);console.log("Done") | node

$
```

**Comment by alexlamsl** [comment]
Pure speculation - this may be a regression caused by the fix for #390 & https://github.com/Microsoft/ChakraCore/issues/2751

**Comment by MSLaguana** [comment]
I think that I fixed this in upstream chakracore a couple of days ago; not sure if the change has made it into a nightly build yet. Thanks for reporting it though!

**Comment by MSLaguana** [comment]
Just confirmed; the fix is in the chakracore-master branch here, but nightlies are currently built from master which is tracking a previous branch in upstream chakracore. This will be changing in the near future I think, so it should be resolved at that time.

**Comment by alexlamsl** [comment]
Thanks for the information (& the fix) – shall I close this since it's already fixed upstream?

**Comment by MSLaguana** [comment]
I'm inclined to leave it open until the fix makes it into a build somewhere.

**Comment by alexlamsl** [comment]
Still an issue as of `v10.0.0`

**Comment by alexlamsl** [comment]
Still an issue as of `v10.1.0`

**Comment by alexlamsl** [comment]
Fixed in `v10.6.0` - thanks!
